src/system.rs

```diff
@@ -176,6 +573,34 @@ impl Docker {
         self.process_into_value(req).await
     }
 
+    /// ---
+    ///
+    /// # Info
+    ///
+    /// Returns Docker client and server information that is running.
+    ///
+    /// # Returns
+    ///
+    ///  - [Info](system/struct.Info.html), wrapped in a Future.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// # use bollard::Docker;
+    /// # let docker = Docker::connect_with_http_defaults().unwrap();
+    /// docker.info();
+    /// ```
+    pub async fn info(&self) -> Result<Info, Error> {
```

Comment:
Thanks for the PR... Can you switch the type here to the one generated from the openapi documentation? https://github.com/fussybeaver/bollard/blob/master/codegen/target/generated-sources/src/models.rs#L4636-L4637